code/modules/mob/living/carbon/emote.dm

```diff
@@ -11,7 +11,29 @@
 	key_third_person = "blinks"
 	message = "blinks."
 
-/datum/emote/living/carbon/blink_r
+/datum/emote/living/carbon/blink/run_emote(mob/living/carbon/human, params, type_override, intentional)
+	. = ..()
+
+	// Don't blink. Not even for a second.
+	for(var/obj/structure/statue/peanut in view(8, human))
+		if (prob(99))
```

Comment:
put the prob before the view() call

Reply:
that'd make every statue in range dunk on you all at once--i'd prefer to have each statue individually maybe jump on you

Reply:
do it anyways.
Just break early when it teleports behind a human once.

Reply:
ok